README.md

```diff
@@ -1 +1,5 @@
 # Hello world
+
+Hello World again
+
+Argg........
```

Comment:
Is this really a good idea?

How about being nice to the user:

Try something like:

    foo.bar();